projects/storefrontapp-e2e-cypress/cypress/e2e/regression/scrolling/scroll-position-restoration.e2e.cy.ts

```diff
@@ -16,24 +16,35 @@ context('scroll Position Restoration', () => {
       },
     }).as('getPage');
 
+    cy.intercept({
+      method: 'GET',
+      pathname: `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
+        'BASE_SITE'
+      )}/cms/pages`,
+      query: {
+        pageType: 'CategoryPage',
+      },
+    }).as('getCategoryPage');
+
     cy.visit('/');
 
     cy.log('Go to category page');
     cy.get('cx-category-navigation a').eq(0).click();
 
     cy.get('cx-product-list-item').should('exist');
-    cy.get('cx-product-list-item .cx-product-name')
+    cy.get('cx-product-list-item .cx-product-name a')
       .eq(3)
       .then(($productItem) => {
         const productName = $productItem.text();
-        cy.wrap($productItem).scrollIntoView().click();
+        cy.wrap($productItem).scrollIntoView().click({ force: true });
```

Comment:
I'm not sure that using `force` here is the best solution because we should be asserting that products are actually visible before we click them. Especially since this test is relating to scroll position, I think that it matters.